website/docs/r/deployment.html.markdown

```diff
@@ -160,6 +160,7 @@ For more info see [Kubernetes reference](http://kubernetes.io/docs/user-guide/an
 * `restart_policy` - (Optional) Restart policy for all containers within the pod. One of Always, OnFailure, Never. For more info see [Kubernetes reference](http://kubernetes.io/docs/user-guide/pod-states#restartpolicy).
 * `security_context` - (Optional) SecurityContext holds pod-level security attributes and common container settings. Optional: Defaults to empty
 * `service_account_name` - (Optional) ServiceAccountName is the name of the ServiceAccount to use to run this pod. For more info see http://releases.k8s.io/HEAD/docs/design/service_accounts.md.
+* `share_process_namespace` - (Optional) Share a single process namespace between all of the containers in a pod. When this is set containers will be able to view and signal processes from other containers in the same pod, and the first process in each container will not be assigned PID 1. HostPID and ShareProcessNamespace cannot both be set. Optional: Defaults to `false`.".
```

Comment:
There is an extra quotation mark (") at the end. Also, Optional is mentioned in the beginning so no need for it in the last phrase.

Reply:
Fixed